app/liquid_tags/liquid_tag_base.rb

```diff
@@ -37,7 +48,7 @@ def initialize(_tag_name, _content, parse_context)
       parse_context.partial_options[:user],
       self,
       :initialize?,
-      policy_class: LiquidTagPolicy,
+      policy_class: parse_context.partial_options.fetch(:policy) { LiquidTagPolicy },
```

Comment:
reading note: Because fetch yields to the provided block when the key is absent - this is a no-op for the default case